nodes/src/nodes/index_search/elasticsearch_store.py

```
@@ -103,9 +103,7 @@ def __init__(self, provider: str, connConfig: Dict[str, Any], bag: Dict[str, Any
 
         similarity = config.get('similarity', 'cosine')
         if similarity not in ('cosine', 'l2_norm', 'dot_product'):
-            raise Exception(
-                'Similarity must be one of: cosine, l2_norm, dot_product'
-            )
+            raise Exception('Similarity must be one of: cosine, l2_norm, dot_product')
```

Ruff (0.15.7)

[warning] Create your own exception

(TRY002)